recipe/meta.yaml

```
@@ -89,7 +92,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
```

```suggestion
        - {{ pin_subpackage({{ name }}, min_pin='x.x', max_pin='x.x') }}
```
Still learning about multi-output recipes, but should this `name` be the variable `{{ name }}`?

---

It's already in a jinja context, as argument of jinja function pin_subpackage.

---

Definitely only saw parentheses, sorry!